login/core/validations/SignInValidation.js

```diff
@@ -0,0 +1,14 @@
+const mailValidation = require('./MailValidation.js');
+const passValidation = require('./PasswordValidation.js');
+const loginValidation = require('./LoginValidation.js');
+const CreateUser = require('../../db/UserResponseServices.js');
+
+
+function SignInValidation(loginRequest, response) {
+  if ((mailValidation.MailValidation(loginRequest.login) || loginValidation.LoginValidation(loginRequest.login)) && passValidation.PasswordValidation(loginRequest.password)) {
+    CreateUser.SignIn(loginRequest, response);
+  } else {
+    response.send('Данные введены неправильно');
```

Comment:
Is this response sending the 400 Bad Request status code or is it going to be a 200 OK but with this message?

Reply:
Ok,i will add requests statuses